features/steps/ubuntu_advantage_tools.py

```diff
@@ -123,6 +123,17 @@ def when_i_install_uat(context, machine_name=SUT):
             )
 
 
+@when("I ensure -proposed is not enabled anymore")
+def when_i_ensure_proposed_not_enabled(context, machine_name=SUT):
+    if context.pro_config.install_from is InstallationSource.PROPOSED:
+        when_i_run_command(
```

Comment:
I think either this should read `I disable the proposed pocket`, or it should run `apt update` right after - to actually ensure no proposed data is in the lists cache.